test/functional/page_objects/dashboard_page_controls.ts

```diff
@@ -255,7 +259,15 @@ export class DashboardPageControls extends FtrService {
     if (additionalSettings) {
       if (controlType === OPTIONS_LIST_CONTROL) {
         // only options lists currently have additional settings
-        await this.optionsListSetAdditionalSettings(additionalSettings);
+        await this.optionsListSetAdditionalSettings(
+          additionalSettings as OptionsListAdditionalSettings
+        );
+      }
+      if (controlType === RANGE_SLIDER_CONTROL) {
```

Comment:
```suggestion
      else if (controlType === RANGE_SLIDER_CONTROL) {
```
Either that, or maybe making this a switch statement would be even cleaner? 🤔